packages/expect/src/__tests__/__snapshots__/matchers.test.js.snap

```diff
@@ -587,462 +587,462 @@ Received: <red>undefined</>"
 exports[`.toBeGreaterThan(), .toBeLessThan(), .toBeGreaterThanOrEqual(), .toBeLessThanOrEqual() equal numbers: [-Infinity, -Infinity] 1`] = `
 "<dim>expect(</><red>received</><dim>).</>not<dim>.toBeGreaterThanOrEqual(</><green>expected</><dim>)</>
 
-Expected: <green>-Infinity</>
+Expected value to be greater than or equal: <green>-Infinity</>
 Received: <red>-Infinity</>"
 `;
 
 exports[`.toBeGreaterThan(), .toBeLessThan(), .toBeGreaterThanOrEqual(), .toBeLessThanOrEqual() equal numbers: [-Infinity, -Infinity] 2`] = `
 "<dim>expect(</><red>received</><dim>).</>not<dim>.toBeLessThanOrEqual(</><green>expected</><dim>)</>
 
-Expected: <green>-Infinity</>
+Expected value to be less than or equal: <green>-Infinity</>
```

Comment:
Here's an example of a test with an improper output after the changes

The code for the test is:

```js
expect(-Infinity).not.toBeLessThanOrEqual(-Infinity);
```

the test fails _because they are equal_, but the output says:

> Expected value to be less than or equal: -Infinity

It should say something link:

> Expected value **not** to be less than or equal: -Infinity